warp/service/backend.go

```diff
@@ -1,7 +1,7 @@
 // (c) 2023, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
-package warp
+package service
```

Comment:
We usually use `service` as the name of an API service, so I think putting the actual Warp Backend here is an outlier.

Could we either move this to a the warp package directly inside of `warp/` or move both the warp backend and the service/client to the same `warp/` package, so they're all just in warp rather than `warpService` ? I'm leaning towards the latter.